shared/types/payment.ts

```diff
@@ -29,6 +29,7 @@ export type CompletedPaymentMeta = {
   submissionId: string
   transactionFee: number
   receiptUrl: string
+  hasS3ReceiptUrl: boolean
```

Comment:
/nit: maybe something like `isReceiptStoredInS3` is more informative (hasS3ReceiptUrl suggests the url is stored in our DB for retrieval)

Reply:
renamed to `hasReceiptStoredInS3` in 69002e5e1